static/src/javascripts/projects/common/modules/ui/subscription-banner.js

```diff
@@ -64,6 +64,31 @@ const fiveOrMorePageViews = (currentPageViews: number) => currentPageViews >= 5;
 const closedAt = (lastClosedAtKey: string) =>
     userPrefs.set(lastClosedAtKey, new Date().toISOString());
 
+const wasBannerClosedBeforeRedeployDate = (
+    bannerRedeploymentDate: number,
+    lastClosedAt: string
+) => {
+    const lastClosedAtTime = new Date(lastClosedAt).getTime();
+    return bannerRedeploymentDate - lastClosedAtTime > 0;
+};
+
+const shouldRedeploy = () => {
+    const bannerRedeploymentDate = new Date(2019, 11, 2, 5, 0).getTime(); // 2 Dec 2019 @ 5:00
+    const lastClosedAt = userPrefs.get(SUBSCRIPTION_BANNER_CLOSED_KEY);
+    const now = new Date().getTime();
+
+    if (now < bannerRedeploymentDate) {
+        return !hasUserAcknowledgedBanner(MESSAGE_CODE);
+    }
+
+    return !lastClosedAt
+        ? true
+        : wasBannerClosedBeforeRedeployDate(
```

Comment:
isn't it simpler to check if the banner was closed **after** redeploy

Reply:
You should also be able to simplify this conditional to `return !lastClosedAt || wasBannerClosedBeforeRedeployDate(...)`